examples/react-native/src/hooks/useApolloClient.ts

```diff
@@ -27,6 +27,8 @@ export const useApolloClient = () => {
       const cache = new InMemoryCache();
       let newPersistor = new CachePersistor({
         cache,
+        // https://github.com/apollographql/apollo-cache-persist/issues/426
+        // @ts-ignore
```

Comment:
I think we should add any to any storage interface definition rather than ts-ignore example.